rply/parsergenerator.py

```diff
@@ -175,44 +177,28 @@ def build(self):
         g.compute_first()
         g.compute_follow()
 
-        # win32 temp directories are already per-user
-        if os.name == "nt":
-            cache_file = os.path.join(
-                tempfile.gettempdir(),
-                "rply-%s-%s-%s.json" % (
-                    self.VERSION, self.cache_id, self.compute_grammar_hash(g)
-                )
-            )
-        else:
-            cache_file = os.path.join(
-                tempfile.gettempdir(),
-                "rply-%s-%s-%s-%s.json" % (
-                    self.VERSION,
-                    os.getuid(),
-                    self.cache_id,
-                    self.compute_grammar_hash(g)
-                )
+        cache_dir = AppDirs("rply").user_cache_dir
+        cache_file = os.path.join(
+            cache_dir,
+            "rply-%s-%s-%s.json" % (
```

Comment:
You probably don't need to start this filename with "rply" since it's going to be in a directory called "rply", but it doesn't hurt to have it either.